### PR TITLE
v2v: add the case of converting ova image with folder ends with '/'

### PR DESCRIPTION
**Pull request by vwu-vera**
Check bz#1964324 and case RHEL-247882.

### CODE REVIEW

**Comment by vwu-vera** [comment]
Cases are passed except "to libvirt" one which has the bug on rhel9.

type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.linux.input_mode.ova.normal.with_dir_end.output_mode.rhev.rhv_upload: PASS (681.37 s)

type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.linux.input_mode.ova.normal.with_dir_end.output_mode.rhev.rhv:PASS (496.94 s)

type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.linux.input_mode.ova.normal.missing_file.output_mode.libvirt

File: v2v/tests/cfg/convert_from_file.cfg [hunk 1]
```diff
@@ -91,6 +91,11 @@
                                     msg_content = 'manifest has a checksum for non-existent file'
                                     expect_msg = yes
                                     skip_vm_check = yes
+                                - with_dir_end:
+                                    ova_dir = OVA_DIR_RHEL_LINUX_V2V_EXAMPLE
+                                    ova_copy_dir = 'OVA_DIR_COPY_V2V_EXAMPLE/OVA_DIR_DIRECTORY_LINUX_RHEL'
+                                    input_file = '${ova_copy_dir}/'
+                                    skip_vm_check = yes
```
Comment (xiaodwan) [inline v2v/tests/cfg/convert_from_file.cfg:98]: if no special reasons, like the guest cannot bootup, the guest cannot be connected, the guest is encryped, etc, 'skip_vm_check = yes' should not be set. And if set, a skip reason should be added.

**Comment by vwu-vera** [comment]
@xiaodwan Removed the line "skip_vm_check = yes" for no reasons.

Please help to review. Thanks.

**Comment by xiaodwan** [comment]
@vwu-vera looks fine, please update the test result, thanks.

**Comment by vwu-vera** [comment]
(1/1) type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.linux.input_mode.ova.normal.with_dir_end.output_mode.rhev.rhv: PASS (664.34 s)

**Comment by vwu-vera** [comment]
@xiaodwan please check the run result. Thanks.

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**